src/scripts/backup-kmq-database.ts

```diff
@@ -0,0 +1,20 @@
+import fs from "fs";
+import { exec } from "child_process";
+import { friendlyFormattedDate } from "../helpers/utils";
+
+const databaseBackupDir = `${process.env.AOIMIRAI_DUMP_DIR}/kmq_backup`;
+
+async function backupKmqDatabase() {
```

Comment:
You can make this return a Promise that resolves on `exec`'s callback, so we can still await the call